index.html

```diff
@@ -37,10 +37,119 @@ <h1>I'm Jiachun, Glad to see you</h1>
         </div>
         
     </section>
-    <section class="works">
-        <div class="container works-text">
+    <section class="cards-container">
+        <div class="card">
+            <img class="card-image" src="./img/portfolio1.jpg" alt="webpage image">
+            <h2>Tonic</h2>
+            <div class="tags">
+                <p>CANOPY</p>
+                <img class="bullet" src="./img/Counter.png" alt="bullet">
+                <span>Back End Dev</span>
+                <img class="bullet" src="./img/Counter.png" alt="bullet">
+                <span>2015</span>
+            </div>
+            <p class="description">
+                A daily selection of privately personalized reads. No accounts or 
+                sign-ups required.
+            </p>
+            <ul class="technologies">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+            </ul>
+            <div>
```

Comment:
- [ ] Please use an indentation of two spaces in all your documents. This is a convention and also best practice.